styles.css

```diff
@@ -2,12 +2,678 @@ body,
+  line-height: 21px;
+}
+
+.get-started-background > section > div > h3 {
+  text-align: justify;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 4vh;
+}
+
+.form-message {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+}
+
+.yours {
+  border: none;
+}
+
+.yours label {
+  width: 80%;
+  color: #7d7c84;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 15px;
+  padding-top: 5px;
+  text-align: start;
+  margin-bottom: 1vh;
+  padding-left: 5%;
+}
+
+.yours input,
+.yours textarea {
+  width: 80%;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 15px;
+  padding-top: 5%;
+  padding-bottom: 5%;
+  padding-left: 10%;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-top: 1px solid rgb(173, 171, 171);
+  border-right: 1px solid rgb(173, 171, 171);
+  border-bottom: 1px solid rgb(173, 171, 171);
+  border-left: 1px solid rgb(173, 171, 171);
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+#getting-started-form > .form-message > div:nth-child(3) {
+  grid-column: 1 / 3;
+  grid-row: 2;
+  margin-top: 1.5vh;
+  margin-right: 0;
+  padding-left: 2%;
+  padding-right: 0;
+}
+
+.yours textarea {
+  height: 100px;
+}
+
+#getting-started-form > .form-message > div:nth-child(3) > textarea {
+  margin: 0;
+}
+
+#getting-started-form {
+  text-align: center;
+}
+
+::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(29, 28, 28);
+  text-align: start;  /* Firefox */
+}
+
+::-ms-input-placeholder { /* Microsoft Edge */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(0, 0, 0);
+  text-align: start;
+}
+
+:-ms-input-placeholder { /* Internet Explorer 10-11 */
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(0, 0, 0);
+  text-align: start;
+}
+
+.button-get-started {
+  width: 50%;
+  background: #1a2236;
+  color: #fff;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  border: 1px hidden;
+  margin-top: 5%;
+  margin-bottom: 50px;
+}
+
+.button-get-started:hover {
+  background: #fff;
+  color: #1a2236;
+  font-weight: 500;
+  border: solid 1px #1a2236;
+}
+
+.button-get-started:active {
+  background: #1a2236;
+  color: #fff;
+}
+
+.footer {
+  display: flex;
+  background: #1a2236;
+  color: #98dede;
+  align-items: center;
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
+
+.footer h2 {
+  margin: 0;
+  padding-left: 10vw;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.icon-sosmed {
+  display: flex;
+  flex-direction: row;
+  gap: 5vw;
+  width: 24px;
+  height: 24px;
+  padding-left: 20vw;
+}
+
+/* start about section desktop version */
```

Comment:
Please don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional 👍

Reply:
remove comment code, to keep codebase clean, fixes